Makefile

```diff
@@ -1,3 +1,5 @@
+export PATH := $(realpath ./scripts/travis/thrift-release/linux-x86_64):$(PATH)
```

Comment:
`realpath` (part of GNU coreutils) is not installed on OS X by default:
```
$ realpath
-bash: realpath: command not found
```

I would use `$(echo $PWD)/scripts/travis/thrift-release/linux-x86_64:$(PATH)` or whatever.